src/components/generic/movieModal/style.tsx

```diff
@@ -69,6 +67,18 @@ export const ModalContent = styled(Content)`
   padding: 10px 40px;
   border-top-left-radius: 25px;
   border-top-right-radius: 25px;
+
+  @media only screen and (min-width: 320px) {
+    flex-direction: column;
+  }
+  @media only screen and (min-width: 768px) {
+    flex-direction: row;
+    justify-content: flex-start;
+  }
+  @media only screen and (min-width: 1200px) {
+    flex-direction: row;
+    justify-content: flex-start;
+  }
```

Comment:
```suggestion
  @media only screen and (min-width: 768px) {
    flex-direction: row;
    justify-content: flex-start;
  }
```

1 is enough for this case.